# catch

*promise.catch(rejectedFn)*

Atajo para manejar promesas rechazadas (rejected)

<code><pre>
promise.catch(error => {
  console.log("Promesa rechazada ðŸ˜¢", error);
});
</pre></code>
